boards/arm/samv7/common/src/sam_progmem_common.c

```diff
@@ -196,7 +184,7 @@ static int init_ota_partitions(void)
 
   return ret;
 }
-#endif
+#endif /* CONFIG_SAMV7_PROGMEM_MCUBOOT_PARTITION */
```

Comment:
We need to stick to one of the names either `OTA_PARTITION` or `MCUBOOT_PARTITION`.

Reply:
There is no intention to change names right now. I missed in rebase, tks!